src/promptflow/promptflow/_core/tools_manager.py

```diff
@@ -90,6 +108,7 @@ def collect_package_tools(keys: Optional[List[str]] = None) -> dict:
                 importlib.import_module(m)  # Import the module to make sure it is valid
                 tool["package"] = entry_point.dist.metadata["Name"]
                 tool["package_version"] = entry_point.dist.version
+                try_to_set_input_index_for_tool(tool)
```

Comment:
please replace with a better function name to explain why need to set input index?

Reply:
Changed to `try_to_set_tool_input_index_for_ui_to_keep_order`, thanks.